toolkit/partner-chains-cli/src/config.rs

```diff
@@ -501,6 +501,15 @@ pub(crate) mod config_fields {
 			_marker: PhantomData,
 		};
 
+	pub(crate) const CARDANO_DATA_SOURCE: ConfigFieldDefinition<'static, String> =
```

Comment:
I would not touch wizards before we have dolos working and plug it here as the last thing in the whole Dolos story.

Now we add inputs and outputs but it can really have one value only.

Regarding implementation: I would go with enum and SelectOptions trait like we have for Http/Https

Reply:
Agree. Implement default behavior for pc-cli, until alternative data source won't be fully intoroduced. Made a note for myself - to implement handling as SelectOptions (when it will be).